src/hooks/usePhotoDate.js

```diff
@@ -0,0 +1,14 @@
+const apiKey = "egwuPjIf803mZ9oZJqCH2Gh36n5nEAyqs9TEIXV4";
+
+export const usePhotoDate = async (date) => {
```

Comment:
Since both usePhoto and usePhotoDate accomplish the same task, we can avoid repeating ourselves and have only one hook like "useFetch", to which we can pass the url that we wish to use. useFetch(url)